src/transport/endpoint.cpp

```
@@ -12,9 +12,11 @@
  ********************************************************************************/
 
 #include "transport/endpoint.h"
-#include <regex>
 #include <sstream>
 #include <functional>
+#include <cstdlib>
+#include <cstring>
```

Cppcheck (2.19.0)

[information] Include file

(missingIncludeSystem)